.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,18 +2,27 @@
 
 ## PR Checklist
 
-- [ ] Has Pytest style unit tests
-- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
-- [ ] New features are documented, with examples if plot related
-- [ ] Documentation is sphinx and numpydoc compliant
-- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
-- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way
+- [ ] Has Pytest style unit tests (and `pytest lib/matplotlib/tests` passes)
+- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant (run
+  `flake8` on changed files to check)
+- [ ] New features (if any) are documented, with examples if plot related
+- [ ] Documentation is sphinx and numpydoc compliant (the docs should
+  [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs)
+  without error)
+- [ ] New documentation conforms to matplotlib style conventions. (If you have
+  `flake8-docstrings` and `pydocstyle<4` installed, run
```

Comment:
I think we can lift the restriction on `pydocstyle<4`. This was introduced in #14710 because of  https://gitlab.com/pycqa/flake8-docstrings/issues/36 but is likely not neccessary anymore.

Reply:
That bug was fixed, but the latest versions (I think `pydocstyle>5.0.1`) now crash on `figure.py`, as per `https://github.com/PyCQA/pydocstyle/issues/437`. `4.0` also crashes and I didn't feel like chasing versions around so I left it at the one that I knew already worked...